features/salt_minions_page.feature

```diff
@@ -6,9 +6,6 @@ Feature: Explore the Minions page
   As an authorized user
   I want to verify all the minion key management features in the UI
 
-  Background:
-    Given I am authorized as "testing" with password "testing"
-
   Scenario: Delete sles-minion system profile (Explore the Minions page)
     Given I am on the Systems overview page of this "sle-minion"
```

Comment:
Does this scenario need also the "Given I am authorized as "testing" with password "testing" step? Like all other scenarios?